cpp/ql/src/semmle/code/cpp/models/implementations/Accept.qll

```diff
@@ -46,8 +46,8 @@ private class Accept extends ArrayFunction, AliasFunction, TaintFunction, SideEf
     i = 1 and buffer = false
   }
 
-  override ParameterIndex getParameterSizeIndex(ParameterIndex i) { i = 1 and result = 2 }
-
+  // NOTE: The size parameter is a pointer to the size. So we can't implement `getParameterSizeIndex` for
+  // this model.
```

Comment:
Then is `hasArrayWithVariableSize` wrong as well?

Reply:
Good point. Fixed in 585606a9331ea97141a62e771080fcfdd1e601bf.